geekbrains/Seminar9/Homework/calc_bot/calc_bot.py

```diff
@@ -0,0 +1,89 @@
+from typing import Union, List
+from functools import wraps
+import os
+import logging
+
+from telegram import Update, constants
+from telegram.ext import filters, ApplicationBuilder, CommandHandler, MessageHandler, ContextTypes
+
+import calc
+
+if __name__ == '__main__':
+    TOKEN = os.environ["TELEGRAM_BOT_TOKEN"]
+    app = ApplicationBuilder().token(TOKEN).build()
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+    level=logging.INFO,
+    filename='log.txt'
```

Comment:
У вас же вроде не Windows платформа, просто у меня кириллица без этого не записывается.
```suggestion
    filename='log.txt',
    encoding='utf-8'
```